docs/Developers/Newcomers_Guide.rst

```diff
@@ -141,6 +141,14 @@ Now you can start working on it.
 
 .. note::
 
+    As stated before, you should never work on an issue without any
+    assignment. Fortunately, cobot is here to help you! So, if you are
+    interested in picking up an issue just write in the gitter chat the
+    following command:
+
+        ``cobot assign <issue_link>``
```

Comment:
Thanks for the advice. I try to figure out if there is something else wrong :) before i commit again